test/test_matmul_scheduler.cpp

```diff
@@ -583,61 +2395,61 @@ TEST_F(MatmulSchedulerTest, EpilogueBeta_CUDA) {
 
   FusionExecutorCache executor_cache(std::move(fusion));
 
-  const int M = 504, N = 136, K = 1024;
+  const int M = 504, N = 136, K = 248;
 
   at::manual_seed(0);
-  const double beta = 2.5;
   auto t0 = matmulAtInput(layout, TensorMatmulPos::A, at::kHalf, M, N, K);
   auto t1 = matmulAtInput(layout, TensorMatmulPos::B, at::kHalf, M, N, K);
-  auto t2 = matmulAtInput(layout, TensorMatmulPos::C, at::kHalf, M, N, K);
+  auto t2 = matmulAtInput(layout, TensorMatmulPos::Bias, at::kHalf, M, N, K);
 
   auto t3 = atMatmul(t0.to(at::kFloat), t1.to(at::kFloat), layout);
+  auto t4 = t3.to(at::kHalf);
+  auto t5 = atBiasEpilogue(t4, t2);
+  auto t6 = at::gelu(t5);
 
-  auto t4 = at::mul(t2, beta).to(at::kFloat);
-  auto t5 = at::add(t3, t4);
-
-  auto outputs = executor_cache.runFusionWithInputs({t0, t1, t2, beta});
+  auto outputs = executor_cache.runFusionWithInputs({t0, t1, t2});
 
   NVF_CHECK(
       !executor_cache.getMostRecentKernelRuntime()->isSegmented(),
       "segmentation did happen");
 
   // NOTE: increasted absolute tolerance to silence false negative verification
   //       caused by different way of calculating reference
-  NVF_CHECK(outputs[0].allclose(t5, 0.01, 0.04));
+  NVF_CHECK(outputs[0].allclose(t6, 0.01, 0.001));
 }
 
-// Matmul test that relies on segmenter for fusion for Ampere:
-//  D = alpha * (A x B) + beta * C
-TEST_F(MatmulSchedulerTest, EpilogueAlphaBeta_CUDA) {
+// Matmul test that uses segmenter for fusion for Ampere:
+//  D = gelu((A x B) + bias)
+//  Precisions:
+//   - A (bf16 - t)
+//   - B (bf16 - t)
+//   - math (fp32 - s)
+//   - D (bf16 - t)
+//   - bias (bf16 - t)
+TEST_F(MatmulSchedulerTest, DISABLED_EpilogueBiasGeluTST_CUDA) {
```

Comment:
Maybe TSS?

Reply:
Added in the second commit.

Thanks for suggestion!